src/pages/components/date-picker/style.mdx

```diff
@@ -60,19 +43,46 @@ If a `min date` is set on the calendar, then any text appearing before that date
 
 **Disabled:** Disabled state should have a `.not-allowed` cursor on hover.
 
+### Calendar menu
+
+| Class              | Property         | Color token                                |
+| ------------------ | ---------------- | ------------------------------------------ |
+| `calendar`         | background-color | `$ui-01`                                   |
+| `calendar`         | box-shadow       | `0 12px 24px 0` <br/> `rgba(0, 0, 0, 0.1)` |
+| `.today`           | background-color | `$interactive-01`                          |
```

Comment:
```suggestion
| `today`            | background-color | `$interactive-01`                          |
```

should the other class names include the period or does this one need to be removed?

Reply:
Yeah that's a good point, we really need to look into this across the board on the stye tab pages and figure out how we want to show these classnames. We do it pretty inconsistently, sometimes it has the full name somethings just the last unique bit.